app/workers/queue_listener.rb

```diff
@@ -28,12 +28,32 @@ def update_pages(params)
     data = params[:params]
     data.delete(:name)
 
-    client.update_petition_page( data.merge(id: petition_id(params) ))
-    client.update_donation_page( data.merge(id: donation_id(params) ))
+    client.update_petition_page(
+      data.merge({
+        id: petition_id(params)
+      }.tap do |load|
+          load[:title] = suffix_title(data[:title], 'petition') if data[:title]
```

Comment:
the only required param is the page id. we can't know what's going to be sent to the worker.